docs/sources/flow/reference/components/otelcol.connector.grafanacloud.md

```diff
@@ -0,0 +1,97 @@
+---
+aliases:
+  - /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.connector.grafanacloud/
+  - /docs/grafana-cloud/send-data/agent/flow/reference/components/otelcol.connector.grafanacloud/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.connector.grafanacloud/
+description: Learn about otelcol.connector.grafanacloud
+labels:
+  stage: experimental
+title: otelcol.connector.grafanacloud
+---
+
+# otelcol.connector.grafanacloud
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" version="<AGENT_VERSION>" >}}
+
+`otel.connector.grafanacloud` accepts span data from other `otelcol` components and generates usage
+metrics.
```

Comment:
```suggestion
`otel.connector.grafanacloud` accepts span data from other `otelcol` components and generates usage metrics.
```

Don't need the hard return at 80 chars... bring sentence back together in one line